scripts/itkdev-docker-compose

```diff
@@ -104,6 +105,26 @@ EOF
   fi
 }
 
+function enableNFS {
+  echo "Setting up nfs..."
+  U=`id -u`
+  G=`id -g`
+
+  LINE="/System/Volumes/Data/Users -alldirs -mapall=$U:$G localhost"
+  FILE=/etc/exports
+  sudo touch $FILE
+  grep -qF -- "$LINE" "$FILE" || sudo echo "$LINE" | sudo tee -a $FILE > /dev/null
+
+  LINE="nfs.server.mount.require_resv_port = 0"
+  FILE=/etc/nfs.conf
+  grep -qF -- "$LINE" "$FILE" || sudo echo "$LINE" | sudo tee -a $FILE > /dev/null
+
+  echo "Restarting nfsd..."
+  sudo nfsd restart
+
+  echo "You should now restarting docker to ensure NFS mounts works..."
```

Comment:
*You should now restart docker....

Reply:
Can we create a command, `itkdev-compose-compose nfs:check`, say, to check that NFS mounts actually work?

Reply:
Don't known it we could try creating a volume... with the NFS driver:

```
docker volume create --driver local \
    --opt type=nfs \
    --opt o=addr=addr=host.docker.internal,rw,nolock,hard,nointr,nfsvers=3 \
    --opt device=:/Users \
    itkdev-composer-test
```

Use the exit code if `0 ` run `docker volume delete itkdev-composer-test`

Reply:
Create volume don't fail before the volume is used...

Reply:
Let's skip this for now.

Reply:
Nope, this don't work.... not sure why... it works in docker-compose

Reply:
Mayby `showmount -e | grep /System/Volumes/Data/Users`